src/kata2.booleanToWord.js

```diff
@@ -1,3 +1,8 @@
-const booleanToWord = boolean => {};
-
+const booleanToWord = boolean => {;
+    if (boolean) {
+        return 'Yes'
+    } else {
+        return 'No'
+    }
```

Comment:
Should this return the boolean passed in as a string ("true" or "false") instead of yes or no?
Either way, it could be simplified into a nice one liner using a ternary operator:
```suggestion
    return boolean ? "Yes" : "No"
```